<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PerformerDialog</class>
 <widget class="QDialog" name="PerformerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Новый мастер</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QCheckBox" name="enabledCheckBox">
           <property name="text">
            <string>Разрешен</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QDateEdit" name="birthdayDateEdit">
           <property name="displayFormat">
            <string>dd.MM.yyyy</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="maleRadioButton">
           <property name="text">
            <string>Муж.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="femaleRadioButton">
           <property name="text">
            <string>Жен.</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="nameLabel">
           <property name="text">
            <string>Имя</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="nameLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="phone1Label">
           <property name="text">
            <string>Телефон 1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="phone1LineEdit">
           <property name="inputMask">
            <string>+79999999999</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="phone2Label">
           <property name="text">
            <string>Телефон 2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="phone2LineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="emailLabel">
           <property name="text">
            <string>E-mail</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="emailLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="addressLabel">
         <property name="text">
          <string>Фактический адрес</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="addressPlainTextEdit">
         <property name="plainText">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="passportGroupBox">
         <property name="title">
          <string>Паспорт</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="pseriesLabel">
              <property name="text">
               <string>Серия</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="pseriesLineEdit"/>
            </item>
            <item>
             <widget class="QLabel" name="pnumberLabel">
              <property name="text">
               <string>Номер</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="pnumberLineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="pdepartmentLabel">
              <property name="text">
               <string>Выдан</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="pdepartmentLineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="paddressLabel">
            <property name="text">
             <string>Адрес регистрации</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="paddressPlainTextEdit"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QListWidget" name="officesListWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>6</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Офисы принятия заявок</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="notesLabel">
         <property name="text">
          <string>Примечания</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="notesPlainTextEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>2</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="loginLabel">
           <property name="text">
            <string>Логин</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="loginLineEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="passwordLabel">
           <property name="text">
            <string>Пароль</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="passwordLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="webadminCheckBox">
         <property name="text">
          <string>Личный кабинет</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QPushButton" name="folderPushButton">
       <property name="text">
        <string>Папка...</string>
       </property>
       <property name="icon">
        <iconset resource="kcrm.qrc">
         <normaloff>:/images/folder.png</normaloff>:/images/folder.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okPushButton">
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="kcrm.qrc"/>
 </resources>
 <connections/>
</ui>
